cmd/lotus-shed/state-stats.go

```diff
@@ -259,7 +258,8 @@ func loadChainStore(ctx context.Context, repoPath string) (*StoreHandle, error)
 	}
 
 	tsExec := consensus.NewTipSetExecutor(filcns.RewardFunc)
-	sm, err := stmgr.NewStateManager(cs, tsExec, vm.Syscalls(proofsffi.ProofVerifier), filcns.DefaultUpgradeSchedule(), nil, mds, index.DummyMsgIndex)
+	sm, err := stmgr.NewStateManager(cs, tsExec, vm.Syscalls(proofsffi.ProofVerifier), filcns.DefaultUpgradeSchedule(), nil, mds,
+		nil)
```

Comment:
lots of these, can we just `null` on the end of the line to minimise diff churn?

Reply:
Ey, I agree this is frustrating. Will fix all such occurrences.

Reply:
Fixed.